src/views/RoomSelector.vue

```diff
@@ -116,6 +132,10 @@ export default {
 			type: Boolean,
 			default: false,
 		},
+		endPointApi: {
+			type: String,
+			default: 'room',
+		},
```

Comment:
```suggestion
		listOpenConversations: {
			type: Boolean,
			default: false,
		},
```

Maybe this is better? We don't use the value, so it could be boolean

Reply:
I chose string so we can use RoomSelector for other different end points in the future.